10.9-libcxx/stable/main/finkinfo/graphics/netpbm11-shlibs.info

```diff
@@ -1,7 +1,7 @@
 Package: netpbm11-shlibs
 Version: 10.97.06
 Revision: 1
-Distribution: 10.9, 10.10, 10.11, 10.12, 10.13, 10.14, 10.14.5, 10.15, 11.0, 11.3
+Distribution: 10.9, 10.10, 10.11, 10.12, 10.13, 10.14, 10.14.5, 10.15, 11.0, 11.3, 12.0, 13.0
```

Comment:
If we're merging all the netpbm11 .info files by conditionaling `/usr/bin/python{,3}`, then we can get rid of the Distribution field.